setup.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+from setuptools import setup, find_packages
+from archive_verify import __version__
+
+
+def read_file(fname):
+    return open(os.path.join(os.path.dirname(__file__), fname)).read()
+
+
+try:
+    with open("requirements/prod", "r") as f:
+        install_requires = [x.strip() for x in f.readlines()]
+except IOError:
+    install_requires = []
+
+setup(
+    name='archive-verify',
```

Comment:
It would be nice to move the metadata to a `pyproject.toml` to get rid of this warning:

DEPRECATION: archive-verify is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change. A possible replacement is to enable the '--use-pep517' option. Discussion can be found at https://github.com/pypa/pip/issues/8559

Reply:
ah, will have to look into that